src/renderer/mod.rs

```diff
@@ -521,7 +526,19 @@ pub fn build_window_config(
         }
     }
 
-    #[cfg(not(target_os = "windows"))]
+    #[cfg(target_os = "macos")]
+    {
+        let cmd_line_settings = SETTINGS.get::<CmdLineSettings>();
+        if cmd_line_settings.opengl {
```

Comment:
also, we need some documentation for this

Reply:
Do you mean in code or in document of neovide. I'll add the latter later.

Reply:
I mean in the command-line [documentation](https://neovide.dev/command-line-reference.html)

Reply:
Got it. Will add it later.